content/2-templating/3-loop/angular/colors.component.ts

```diff
@@ -11,3 +12,10 @@ import { Component } from "@angular/core";
 export class ColorsComponent {
   colors = ["red", "green", "blue"];
 }
```

Comment:
@LcsGa great job, thanks! Just a small thing: for consistency, colors should probably be a signal as well!

Reply:
For the rest 💯

Reply:
Hi! You're welcome :grin:!
- This component is in angular v13- => not signal based
- If you take a look at the other file bellow, it's still not a signal and the reason is => colors are not mint to change over time: in this regard, they can stay basic variables